Competition2018/src/main/java/competition/subsystems/lean/LeanSubsystem.java

```diff
@@ -57,13 +70,18 @@ public boolean hitBar() {
 	 * increases leaning arm speed
 	 */
 	public void increaseSpeed() {
-
+		speed = leanSpeed.get();
```

Comment:
if you want to keep this, I suggest doing the same fast/slow as I mentioned before.

Reply:
Would it be necessary to keep the Increase and Decrease Speed methods?